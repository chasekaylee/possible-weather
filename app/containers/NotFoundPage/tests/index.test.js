import React from 'react';
import { render } from 'react-testing-library';

import NotFoundPage from '../index';

describe('<NotFoundPage />', () => {
  it('should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(<NotFoundPage />);
    expect(firstChild).toMatchSnapshot();
  });
});
