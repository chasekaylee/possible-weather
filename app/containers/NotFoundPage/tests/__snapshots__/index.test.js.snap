// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> should render and match the snapshot 1`] = `
<h1>
  Sorry! We could not find the page you were looking for!
</h1>
`;
